content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -59,10 +106,15 @@ resources:
               secret: YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXoxMjM0NTY=
 ```
 
-Each `resources` array item is a separate config and contains a complete configuration. The
-`resources.resources` field is an array of Kubernetes resource names (`resource` or `resource.group`)
+Each `resources` array item is a separate configuration and contains a complete configuration. The
```

Comment:
I know we're elaborating on the resources array item by speaking about individual fields. However, the first sentence feels incomplete to me. Perhaps we should include what it contains a complete configuration of?